wagtail/admin/templates/wagtailadmin/shared/main_nav.html

```diff
@@ -1,18 +1,18 @@
 {% load wagtailadmin_tags %}
 {% load i18n %}
-<nav class="nav-main">
+<nav role="navigation" class="nav-main" aria-label="{% trans "Main navigation" %}">
     <ul>
         {{ menu_html }}
 
-        <li class="footer" id="footer">
+        <li class="footer" id="menu-footer">
```

Comment:
As far as I know the semantics of ids does not have an impact on assistive technology – could you explain this change further?